rpcserver.go

```diff
@@ -470,3 +470,12 @@ func (r *rpcServer) SendPayment(paymentStream lnrpc.Lightning_SendPaymentServer)
 
 	return nil
 }
+
+func (r *rpcServer) ShowRoutingTable(ctx context.Context,
+	in *lnrpc.ShowRoutingTableRequest) (*lnrpc.ShowRoutingTableResponse, error) {
+	rpcsLog.Debugf("[ShowRoutingTable]")
+	// TODO (mkl): implement it as a thread-safe method of RoutingManager. Try to not use RT.String() directly
```

Comment:
I think this TODO should be completed before we merge the package in. The easiest path would likely modifying the `routingMgr`'s string method to grab a `sync.Mutex` during function execution. 

As an aside, I think the `ShowRoutingTableResponse` should be discretized into a more specific proto with (having some named fields) which lead to newly defined protobufs. It's likely that we'll be adding more RPC commands for querying/modifying the internal state of the deamon's routing table. If we more or less transliterate the manager's public structs into proto definitions then we can re-use these components when future RPC commands are implemented.